### ISSUE TITLE
Machine Learning Crash Course with TensorFlow APIs by Google

### ISSUE THREAD

**Issue by digpatel123** [rupali-codes/LinksHub#1697]
### Site URL

https://developers.google.com/machine-learning/crash-course

### Description of the above provided site

It's a good foundational course to start learning Machine Learning.

### Category

Machine Learning

### Sub Category

Framing, Descending into ML, Reducing Loss, First Steps with TF, Generalization, Training and Test Sets, Validation Set, Representation, Feature Crosses, Regularization: Simplicity, Logistic Regression, Classification, Regularization: Sparsity, Neural Networks, Training Neural Nets, Multi-Class Neural Nets, Embeddings, Static vs. Dynamic Training, Static vs. Dynamic Inference, Data Dependencies, Fairness

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

**Comment by rupali-codes** [comment]
Hi @digpatel123, the category is going to be `ai` and subcategory is going to be `machine-learning`

**Comment by digpatel123** [comment]
@rupali-codes The parent link to the course I wished to add is already in the machine-learning category. I am closing this issue for that reason. However, I wish to provide other useful links and courses, for which I will create another issue to contribute.
